turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -2892,7 +2897,11 @@ impl ModulePart {
     }
     #[turbo_tasks::function]
     pub fn export(export: RcStr) -> Vc<Self> {
-        ModulePart::Export(Vc::cell(export)).cell()
+        ModulePart::Export(Vc::cell(export), Vc::cell(false)).cell()
+    }
+    #[turbo_tasks::function]
+    pub fn proxied_export(export: Vc<RcStr>) -> Vc<Self> {
```

Comment:
I used `Vc<RcStr>` because it's created only from `ModulePart::Export`, but not sure if it's ideal.

Reply:
I mean consistency